src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -29,11 +30,26 @@ public void promptNextPlayer(){
         }
     }
 
+    private void printScore(){
+        System.out.println("X wins: " + score[0]  + "    O wins: " + score[1] +
+                    "    Ties:" + score[2]+ "");
+    }
```

Comment:
The feature 2 requirement shows to keep track of how many games were won, lost, tied by each player.
Therefore, you should add a lost score.

Reply:
This is implied by the opponents victories. Will not fix.

Reply:
Fixed in pull request #4